packages/app-desktop/integration-tests/markdownEditor.spec.ts

```diff
@@ -253,5 +255,24 @@ test.describe('markdownEditor', () => {
 		// Note viewer should be focused
 		await expect(noteEditor.noteViewerContainer).toBeFocused();
 	});
+
+	test('local file URLs setting should allow loading images from local file URLs', async ({ mainWindow, electronApp }) => {
+		await setSettingValue(electronApp, mainWindow, 'renderer.fileUrls', true);
+
```

Comment:
The `setSettingValue` function [was taken from this (currently open) pull request](https://github.com/laurent22/joplin/pull/12237).